src/components/UI/GameCard.tsx

```diff
@@ -87,13 +87,15 @@ const GameCard = ({ cover, title, appName, isInstalled, logo }: Card) => {
             filter: isInstalled ? 'none' : `grayscale(${effectPercent})`,
           }}
           className="gameLogo"
+          loading="lazy"
```

Comment:
Good. Did you notice any difference? I'm thinking of a way of caching the images instead of downloading them every time. But I haven't find the best way yet.